package.json

```diff
@@ -44,13 +46,13 @@
     "test": "karma start ./spec/karma.conf.js",
     "build": "npm run rollup && npm run uglify",
     "lint": "eslint .",
-    "lintfix": "npm run lint -- --fix",
```

Comment:
I've removed this script to fix linting issues as it is expected the pre-commit hook will take care of this for us.

Reply:
Sometimes lintfix can not resolve everything and then we need a log what doesn't worked

Reply:
Should be fine as a commit cannot be made when there are unfixable linting errors. For example:

![An example of a lint failing on a commit](https://user-images.githubusercontent.com/695720/165125188-280e7939-0a1a-4605-8502-265021362325.png)

Reply:
Nice 😄 
But I would like to keep it. If I don't want to commit but to fix the issues, I can't do it anymore. I think it doesn't hurt if we keep it.

Reply:
>So then this behavior is fine right? The commit is not actually made unless all issues are resolved, either manually or by auto-fixing.

Yes is fine for the commit part but I meant that we should keep `lintfix`.

Reply:
You can always run `npm run lint -- --fix`. But I don't see a scenario where you need this unless you explicitly opt out of pre-commit checks.

Reply:
> You can always run npm run lint -- --fix.

Sure but we can also simply keep `lintfix`. What is the benefit of removing it? Maybe you are right and it's not needed anymore, but I would like to test it first in practice and check if it is obsolete.

> But I don't see a scenario where you need this

For example if I want to commit and then it failed because of lint issues. Then I fix them but want to make sure that everything is fine without commiting directly, then I need `lintfix`